packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -393,6 +394,32 @@ export default class RouterService extends Service {
     return this._router._routerMicrolib.recognizeAndLoad(internalURL);
   }
 
+  /**
+   * Refreshes all currently active routes, doing a full transition.
```

Comment:
I just copied the documentation text from the RFC. Should I add example code similar to the other methods?